library/src/main/java/com/heinrichreimersoftware/materialintro/slide/SimpleSlide.java

```diff
@@ -34,7 +35,7 @@
     private static final int DEFAULT_PERMISSIONS_REQUEST_CODE = 34; //Random number
     private SimpleSlideFragment fragment;
 
-
+    private final int id;
```

Comment:
I thought about resource ids as default (mostly, you'll just use the title resource id for this imho), altough using simple ints like 1, 2, 3, ... would be fine as well, I don't want to force anyone to create ids for this... But changing it to long is no problem